src/renderer/i18n/en.ts

```
@@ -357,7 +357,14 @@
       titleAsc: "Title (A-Z)",
       titleDesc: "Title (Z-A)",
     },
-    empty: "No projects yet. Create your first project to get started!",
+    empty: {
+      welcome: "Welcome to MulmoCast!",
+      introduction1: "First, click the {createNew} button.",
+      introduction2: "A project with sample content will be created automatically. Then simply press {generateVideo} button to try it out right away.",
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)